engineering_tools/code_coverage/make_with_coverage.bash

```diff
@@ -0,0 +1,72 @@
+
```

Comment:
1. I think the `#!/bin/bash` shebang line needs to be the first in the file, not sure if that's why you're explicitly invoking `bash make_with_coverage.bash` elsewhere in this PR, but with that you should just be able to run it with `./make_with_coverage.bash`.
2 . Right after that line I'd drop in the standard copyright notice on this and the other shell script.